app/components/Analyst/Project/Announcements/AnnouncementsForm.tsx

```diff
@@ -194,6 +195,9 @@ const AnnouncementsForm = ({ query }) => {
   const removeSelfReference = (ccbcList: Array<any>) => {
     return ccbcList.filter((ccbcId) => ccbcId.ccbcNumber !== ccbcNumber);
   };
+  const handleUpdateToastKey = () => {
+    setToastKey((key) => key + 1);
+  };
 
   const handleSubmit = () => {
     hiddenSubmitRef.current.click();
```

Comment:
Nice work on this toast, it looks great.

I think this solution is slightly overcomplicated. I think we can simply set `updatedCcbcItems` to null at the top of the `handleSubmit`?
```suggestion
  const handleSubmit = () => {
    setUpdatedCcbcItems(null);
    hiddenSubmitRef.current.click();
```

Reply:
Worked perfectly! Much better than my convoluted method